frontend/src/my_team/MyTeamView.jsx

```diff
@@ -152,14 +152,16 @@ function MyTeamView() {
             setMyTeamStyle={setStyle}
```

Comment:
<img width="1707" alt="Screenshot 2024-07-30 at 4 29 18 PM" src="https://github.com/user-attachments/assets/e0719fb6-f50b-499c-8732-6961d81c89a0">

Removes the sorting icons from column headers. Sorting the stats would scramble the order of data passed into the line graph, so we don't want to allow the user to do that. There's also no reason to copy the stats into another independently sortable array because there is no additional utility that justifies the extra space cost. Any data trends/ information given by a sorted table is easily - if not better - communicated by the line graph already.